packages/svelte-check/src/index.ts

```diff
@@ -0,0 +1,156 @@
+    const endOffset = offsetAt(diagnostic.range.end, text);
+    const codePrev = chalk.cyan(text.substring(Math.max(startOffset - 10, 0), startOffset));
+    const codeHighlight = chalk.magenta(text.substring(startOffset, endOffset));
+    const codePost = chalk.cyan(text.substring(endOffset, endOffset + 10));
+    const code = codePrev + codeHighlight + codePost;
+    const msg = `${diagnostic.message} ${source}\n${position}\n${chalk.cyan(code)}`;
+
+    if (diagnostic.severity === DiagnosticSeverity.Error) {
+        console.log(`${chalk.red('Error')}: ${msg}`);
+        return 1;
+    }
+
+    console.log(`${chalk.yellow('Warn')} : ${msg}`);
+    return 0;
+}
+
+async function getDiagnostics(workspaceUri: URI) {
+    const clientConnection = await prepareClientConnection();
+
+    const files = glob.sync('**/*.svelte', {
+        cwd: workspaceUri.fsPath,
+        ignore: ['node_modules/**'],
+    });
+    const absFilePaths = files.map((f) => path.resolve(workspaceUri.fsPath, f));
+
+    console.log('');
+    let errCount = 0;
+
+    for (const absFilePath of absFilePaths) {
+        const text = fs.readFileSync(absFilePath, 'utf-8');
+        clientConnection.sendNotification(DidOpenTextDocumentNotification.type, {
+            textDocument: {
+                languageId: 'svelte',
+                uri: URI.file(absFilePath).toString(),
+                version: 1,
+                text,
+            },
+        });
+
+        try {
+            const res = (await clientConnection.sendRequest('$/getDiagnostics', {
+                uri: URI.file(absFilePath).toString(),
+            })) as Diagnostic[];
+            if (res.length > 0) {
+                console.log('');
+                console.log(`${chalk.green('File')} : ${chalk.green(absFilePath)}`);
+                res.forEach((d) => (errCount += logDiagnostic(d, text)));
+                console.log('');
+            }
+        } catch (err) {
+            console.log(err);
+        }
+    }
+
+    return errCount;
+}
+
+(async () => {
+    const myArgs = argv(process.argv.slice(1));
+    let workspaceUri;
+
+    let workspacePath = myArgs['workspace'];
+    if (workspacePath) {
+        if (!path.isAbsolute(workspacePath)) {
+            workspacePath = path.resolve(process.cwd(), workspacePath);
+        }
+        console.log(`Loading Svelte-Check in workspace path: ${workspacePath}`);
+        workspaceUri = URI.file(workspacePath);
+    } else {
+        console.log(`Loading Svelte-Check in current directory: ${process.cwd()}`);
+        workspaceUri = URI.file(process.cwd());
+    }
+
+    console.log('');
+    console.log('Getting Svelte diagnostics...');
+    console.log('====================================');
+    const errCount = await getDiagnostics(workspaceUri);
+    console.log('====================================');
+
+    if (errCount === 0) {
+        console.log(chalk.green(`Svelte-Check found no error`));
```

Comment:
I think it's a bit funny to capitalize `Svelte-Check` in these messages since the package name is lowercase and that's how people will write it on the command line

Also, this would typically be said as "no errors"

Reply:
Good point.